sc-leaderboard-react/pages/api/score-items.ts

```diff
@@ -2,31 +2,25 @@ import { ScoreItem } from "../../lib/definitions";
 import client from "../../lib/mongodb";
 import { NextApiRequest, NextApiResponse } from "next";
 
-const a = async (req: NextApiRequest, res: NextApiResponse<ScoreItem[]>) => {
+const api = async (req: NextApiRequest, res: NextApiResponse<ScoreItem[]>) => {
```

Comment:
Alternative implementation that would probably work is the following (not tested obv.)

**Create /api/score-items/route.ts** with the following code
```js
//.. imports, idk which ones you need
const db = client.db("Summercamp2025"); //might want to store this in a lib, idk

export async function GET(request: Request) {
     const points = await db.collection<ScoreItem>("Points").toArray();
     return Response.json(points);
}

export async function POST(request: Request) {
    //... implementation
}


//.. other implementation n shit
```

See [route handlers](https://nextjs.org/docs/app/building-your-application/routing/route-handlers) for more information

Reply:
I'll remove this whol piece of code in the future. I'm going to use the vercel store with postgres
But will do

Reply:
Also, I'm using pages router and the way you are suggesting is not supported for pages router.